Software/app/freefall_lorawan/README.md

```diff
@@ -0,0 +1,76 @@
+
+Additionally, try to remember that:
+- Setting the debugger will consume more power. The debugger is set in the macro `DEBUGGER_ON` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h).
+- Keeping peripherals, such as the sensors, off by controlling the load switches will reduce power consumption significantly. But, of course, you can not turn the load switch connected to the accelerometer off without disabling the free fall detection as well.
+
+### LoRa extra's
+
+Some extra setup configurations can be done to change the behaviour of the LoRaWAN capabilities. These include:
+1. Setting the activation method (OTAA or ABP) in `LORAWAN_DEFAULT_ACTIVATION_TYPE` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/lora_app.h). OTAA is recommended as it is the most secure.
+2. The data rate can be set in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/lora_app.h). The default configuration uses the ADR. Should you want to set your preferred data rate, set `LORAWAN_ADR_STATE` to `LORAMAC_HANDLER_ADR_OFF` and set `LORAWAN_DEFAULT_DATA_RATE` to your preference. A list of the options per region are shown in [Region.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/lib/STM32WLxx_LoRaWAN/LoRaWAN/Mac/region/Region.h) in the STM32WLxx_LoRaWAN library.
+3. `ACC_FF_LORA_PORT` can be changed in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h), which is used to configure to transmission port. The LoRaWAN keys mentioned in the default section can be altered here as well.
+4. Also in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h), the downlink port can be set by altering `ACC_FF_DOWNLINK_PORT`.
+
+### Debugger
+
+For debugging, the firmware has to support it first. The debugger is set in the macro `DEBUGGER_ON` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h).
+
+## Setup
+
+1. See if your device can detect free fall events. The default values should be enough to trigger on minor drops (try to not break the device) or even sudden movements with your arm whilst holding the GNSE. A message should be transmitted over LoRaWAN containing the counter. If none can be seen, try to debug the device as described in the Observation section.
+2. Test you setup for free fall events and change it to your liking (with the steps in the Configuration chapter). Be aware that `ACC_FF_THRESHOLD` would have to be increased to increase the change of a free fall event whilst `ACC_FF_DURATION` would have to be decreased for a similar effect.
```

Comment:
Link the Configuration section and remove the word "chapter". What does "to increase the change of a free fall event" mean? Please reword this a bit